### ISSUE TITLE
Font loading is slow

### ISSUE THREAD

**Issue by ulugbekna** [ocaml/ocaml.org#797]
Hi! 

Font loading is a bit slow, see the gif below, in which I hard-refresh (cmd+shift+r) the page and you see font changing

https://user-images.githubusercontent.com/16353531/212172417-97b7a452-5fb8-4ff9-8e2e-25c36c4147c1.mov

I'm no expert in this, but I wonder if loading it from some CDN (e.g., google fonts) could both improve the loading speed and decrease server load?

**Comment by cuihtlauac** [comment]
Hi @ulugbekna, thanks for your report. This is indeed annoying. Do you observe the same problem with all browsers? Does it happen every time, for every page?

**Comment by sabine** [comment]
This should happen only once (and on browser-cache-invalidating hard-refresh) since, after that, your browser will cache these fonts. However, there is still an issue in how we handle CSS which forces you to do a hard-refresh of the page when something is off (see https://github.com/ocaml/ocaml.org/issues/704).

Loading from Google Fonts can potentially violate the GDPR by unnecessarily exposing the user's IP address to Google. A different CDN would be viable, and, in the longer-term a good addition.

I believe that time is - at the moment - best spent by fixing the underlying issue that causes the need to hard-refresh the page.

**Comment by sabine** [comment]
Closing in favor of https://github.com/ocaml/ocaml.org/issues/1179
